packages/eslint-config-airbnb-base/rules/imports.js

```diff
@@ -93,8 +93,8 @@ module.exports = {
     // Style guide:
 
     // disallow non-import statements appearing before import statements
-    // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/imports-first.md
-    'import/imports-first': ['error', 'absolute-first'],
+    // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/first.md
+    'import/first': ['error', 'absolute-first'],
```

Comment:
Let's also keep a line that disables `import/imports-first`, in case anyone is overriding it.